src/java/com/github/ruleant/getback_gps/LocationService.java

```diff
@@ -619,7 +619,7 @@ private boolean requestUpdatesFromProvider() {
 
             try {
                 mLocationManager.requestLocationUpdates(
-                        mProviderName,
```

Comment:
Thanks for your suggestion.

The reasoning behind replacing the mProviderName parameter with the PASSIVE_PROVIDER provider, to save energy is a good idea. But just replacing it is a bit simplistic.
If you look at the code base, you'll notice that mProviderName is defined by a method, that takes a lot of parameters into account to choose the best location provider. Low Energy is one of them : https://github.com/ruleant/getback_gps/blob/master/src/java/com/github/ruleant/getback_gps/LocationService.java#L242
The getProvider method will choose the Passive Provider if that is the best match.

That said, the app is never running in background and will only look for a location if it is actually in use. So the number of requests to a location provider will be limited, anyway. At that time the user is using the app they want the most accurate location, instantly. Waiting for another app to request a location will disrupt the user experience.